samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/configuration/BeansBatchConfig.java

```diff
@@ -18,12 +11,20 @@
 import org.springframework.batch.core.repository.support.JobRepositoryFactoryBean;
 import org.springframework.beans.factory.BeanCreationException;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.BeanFactoryPostProcessor;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.DependsOn;
 import org.springframework.transaction.PlatformTransactionManager;
 
 /**
+ * This class contains the configuration like jobLauncher,Jobrepository etc.
+ * @author ssarmoka
+ */

```

Comment:
please consolidate with class javadoc. This doc is currently connected to an import, which is obviously a mistake.